src/Core/Silk.NET.BuildTools/Converters/Readers/OpenCLReader.cs

```diff
@@ -930,19 +945,117 @@ public IEnumerable<Enum> ReadEnums(object obj, BindTask task)
                     }
                 }
             }
+
+
+            var groups = new Dictionary<string, (List<Token> list, bool is64bit, string translatedName)>();
+            foreach (var @enum in allEnums)
+            {
+                var group = @enum.Value.Item2;
+
+                var is64bit = ulongEnums.Contains(group);
```

Comment:
Can we use this data to add a `[Flags]` attribute to the relevant enums? Otherwise you get compiler warnings when doing bitwise stuff.

Reply:
Yeah, I was also thinking about this.
I wonder what the process should be for larger PRs like this: keep adding stuff until it's all good?

Reply:
Generally we do prefer smaller PRs, but in the world of BuildTools this is still only a small PR so your call!